### PR TITLE
Fix: Classes rebuild command doesn't print classes

### LINKED ISSUE #16325: [Bug]: Classes rebuild command doesn't print classes
### Pimcore version

v11.1.2

### Steps to reproduce

Create any object class and execute classes rebuild command:
```bash
bin/console pimcore:deployment:classes-rebuild -vvv --create-classes --delete-classes
```

Line shown in screen below is causing this problem:
image

### Actual Behavior

Output of command:

```
---------------------
Saving all classes
---------------------
Saving all object bricks
---------------------
Saving all field collections
```

### Expected Behavior

```
---------------------
Saving all classes
Category [3] saved
Guarantee [4] saved
Producer [2] saved
Product [1] saved
Vat [5] saved
---------------------
Saving all object bricks
---------------------
Saving all field collections
```

### PR DESCRIPTION
**Pull request by blankse**
## Changes in this pull request  
Resolves https://github.com/pimcore/pimcore/issues/16325

### CODE REVIEW

File: lib/Model/DataObject/ClassDefinition/ClassDefinitionManager.php [hunk 1]
```diff
@@ -29,6 +29,8 @@ class ClassDefinitionManager
 
     /**
      * Delete all classes from db
+     *
+     * @return array<int, array{string, string, string}>
```
Comment (jdreesen) [inline lib/Model/DataObject/ClassDefinition/ClassDefinitionManager.php]: Actually, it is:

```suggestion
     * @return list<array{string, string, string}>
```

File: lib/Model/DataObject/ClassDefinition/ClassDefinitionManager.php [hunk 2]
```diff
@@ -58,16 +60,16 @@ public function cleanUpDeletedClassDefinitions(): array
 
     /**
      * Updates all classes from PIMCORE_CLASS_DEFINITION_DIRECTORY
+     *
+     * @return array<int, array{string, string, string}>
```
Comment (jdreesen) [inline lib/Model/DataObject/ClassDefinition/ClassDefinitionManager.php]: Here, too:

```suggestion
     * @return list<array{string, string, string}>
```

**Comment by kingjia90** [comment]
LGTM, thank you

By looking at the [PR](https://github.com/pimcore/pimcore/commit/e4022d015dff6d15fbf5d7c5f39a6b4150d83ec9) which introduced this regression, noticed that we could remove this line 
https://github.com/pimcore/pimcore/blob/9f9336f0b82e07b5c07d58370d464b9f70e7c5f1/bundles/CoreBundle/src/Command/ClassesDefinitionsBuildCommand.php#L59

Could you please also add this change to this PR?

**Comment by blankse** [comment]
@kingjia90 I removed the line: https://github.com/pimcore/pimcore/pull/16326/commits/953d20bc218714d3ba901c88c4cbb9948ee22b22

**Pull request merged**